src/test/java/com/commercetools/sync/products/helpers/variantreferenceresolver/Utils.java

```diff
@@ -0,0 +1,65 @@
+package com.commercetools.sync.products.helpers.variantreferenceresolver;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.Map;
+
+import static com.commercetools.sync.products.helpers.VariantReferenceResolver.REFERENCE_ID_FIELD;
+import static com.commercetools.sync.products.helpers.VariantReferenceResolver.REFERENCE_TYPE_ID_FIELD;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public final class Utils {
```

Comment:
I'd rather rename this class to easily identify what kind of utilities that holds in this class.

Reply:
https://github.com/commercetools/commercetools-sync-java/commit/fe66ce412fc4bbab07c035d1db0a4ff856d09830